lib/foreman/foreman_url_renderer.rb

```diff
@@ -21,11 +21,6 @@ def default_url_options
       returns String, desc: "Rendered URL"
     end
     def foreman_url(action = nil, params = {}, unescaped_params = {})
```

Comment:
Should we change this to 
```suggestion
    def foreman_url(action = 'provision', params = {}, unescaped_params = {})
```
just to maintain the same behaviour in case someone forgot to update it? @lzap was there any specific reason we didn't just default to it in the first place?

Reply:
I think this was intended:
```suggestion
    def foreman_url(action, params = {}, unescaped_params = {})
```

Reply:
good point

Reply:
This should be changed to what @ekohl suggested